components/form/Quiz.vue

```diff
@@ -336,6 +336,14 @@ export default defineComponent({
         }
         if (hasDuplicates(options.value))
           return openSnackbar("error", "Error.OptionsCannotBeSame");
+        for (let i = 0; i < options.value.length; i++) {
+          if (options.value[i].answer.length > 255) {
+            return openSnackbar("error", t("Error.CannotHaveMoreCharacters", { input: t("Inputs.AnswerOption"), max: 256 }));
+          }
+        }
+        if (form.question.value.length > 255) {
+          return openSnackbar("error", t("Error.CannotHaveMoreCharacters", { input: t("Inputs.Question"), max: 256 }));
+        }
```

Comment:
The maximum length of the question is actually 4096, see https://api.test.bootstrap.academy/challenges/redoc#tag/MultipleChoice/paths/~1tasks~1%7Btask_id%7D~1multiple_choice/post

```suggestion
        if (form.question.value.length > 4096) {
          return openSnackbar("error", t("Error.CannotHaveMoreCharacters", { input: t("Inputs.Question"), max: 4096 }));
        }
```